docs/source/development-guide/getting-started.rst

```diff
@@ -0,0 +1,29 @@
+.. _getting-started:
```

Comment:
I tried to keep this minimal and mostly linked out because most of this stuff is described in other pages, and I want to encourage new users to read from there. The bash should only work for people who already know about poetry and how to use it, everyone else should read the poetry-environment page.